spec/takeaway_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'takeaway'
+require 'menu'
+
+describe Takeaway do
+  let(:x) { {"pizza"=>3, "curry"=>4, "chopsuey"=>2} }
```

Comment:
Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.